### PR TITLE
New GET endpoints of api/v1/overlays

### LINKED ISSUE #3606: New GET endpoints of api/v1/overlays
Branch `3606-overlays_new_endpoints`

Feat. flag will be the same as visualizations (`active_record_vis_endpoint`)

### PR DESCRIPTION
**Pull request by Kartones**
Closes #3606

### CODE REVIEW

**Comment by Kartones** [comment]
@rafatower CR please

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/3659/))

**Comment by Kartones** [comment]
@javisantana CR for FE please

**Comment by Kartones** [comment]
@javierarce CR please, @javisantana is fleeing from the office right now :disappointed:

File: app/controllers/carto/api/overlays_controller.rb [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        head 401 and return unless is_uuid?(params.fetch('id'))
+
+        member = CartoDB::Overlay::Member.new(id: params.fetch('id')).fetch
+        head 401 and return if member.nil?
+
+        vis = Carto::Visualization.where(id: member.visualization_id).first
+        head 403 and return if vis.user_id != current_user.id
+      end
+
+      def check_owner_by_vis
```
Comment (rafatower) [inline app/controllers/carto/api/overlays_controller.rb]: being picky: by_vis_id?

**Comment by rafatower** [comment]
Maybe adding a negative example in tests would be nice, other than that :+1:

**Comment by Kartones** [comment]
Negative specs I'll leave them for the future, I manually tried and as there were no tests I was more focused on building "working specs", but noted.

**Comment by javierarce** [comment]
:+1: the .js looks fine

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/3665/))

**Pull request merged**